content/docs/user-guide/components/reference/physx/shape-collider.md

```diff
@@ -1,93 +1,78 @@
+* You want to use a Shape component such as [Polygon Prism Shape](/docs/user-guide/components/reference/shape/polygon-prism-shape/) that is not provided by PhysX Collider.
+* You have existing Shape components and don't want to migrate them to use PhysX Collider components.
 
-****In Scene Queries****
-When enabled, this shape collider can be queried for raycasts, shapecasts and overlap.
+## Limitations
 
-****Library (Physics Materials)****
-The physics material library of the project.
-
-****Slot (Physics Materials)****
-Choose a physics material for this shape collider.
+The PhysX Shape Collider component has some limitations compared to the PhysX Collider component:
 
-****Tag****
-Set a tag for this shape collider. Tags can be used to quickly identify components in script or code.
+* Only one Shape component can be used per entity, and so only one PhysX Shape Collider component is supported per entity. Any number of PhysX Collider components can also be used on the same entity, however.
+* The position and rotation of the PhysX Shape Collider component can't be offset relative to the entity position.
 
-****Rest offset****
-PhysX bodies come to rest separated by the sum of their rest offset values. The **Rest offset** value must be less than the **Contact offset** value. Valid values rage from **-Infinity** to **50**.
+## Properties 
 
-****Contact offset****
-PhysX bodies generate contacts when they are within the sum of their contact offset values. The **Contact offset** value must be greater than the **Rest offset** value. Valid values rage from **0** to **50**.
+![PhysX Shape Collider component interface](/images/user-guide/components/reference/physx/physx-shape-collider-ui-01.png)
 
-****Draw collider****
-Render this shape collider in the viewport. Enabled by default.
+| Property | Description | Value | Default |
+| - | - | - | - |
+| **Collision Layer** | Assigns the collider to a collision layer. Collision layers can be used to restrict physical interactions between PhysX objects. | Any collision layer defined the project [Collision Layers](/docs/user-guide/interactivity/physics/nvidia-physx/configuring/configuration-collision-layers/).  | `Default` |
+| **Collides With** | Assigns the collider to a collision group. Collision groups contain the collision layers that this collider can collide with. | Any collision group defined the project [Collision Groups](/docs/user-guide/interactivity/physics/nvidia-physx/configuring/configuration-collision-groups/). | `All` |
+| **Trigger** | If enabled, this collider functions as a trigger. A trigger performs a quick overlap test and does not apply forces or return contact point information. Use this to speed-up PhysX computations where a simple overlap test between colliders is sufficient. Triangle meshes are not supported as triggers. | Boolean | `Disabled` |
```

Comment:
A lot of my suggestions regarding properties in the `content/docs/user-guide/components/reference/physx/collider.md` page are also applicable to this page. You can refer back to them if you want to apply those suggestions here as well.

Reply:
Added.